services/base.spec.js

```diff
@@ -15,17 +15,26 @@ const BaseService = require('./base')
 require('../lib/register-chai-plugins.spec')
 
 class DummyService extends BaseService {
-  async handle({ namedParamA }, { queryParamA }) {
+  static render({ namedParamA, queryParamA }) {
```

Comment:
Being on the reading side of this reminded me of @platan's suggestions about this test code made in https://github.com/badges/shields/pull/1589#pullrequestreview-108519067.

I think I implemented all of his suggestions except this one. Would you mind making that change while you're here?

https://github.com/badges/shields/pull/1589#discussion_r178595180